docs/guide/tutorial-docker.md

```diff
@@ -0,0 +1,50 @@
+Yii and Docker
+==============
+
+For development and deployments Yii applications can be run as Docker containers.
+
+You can read more about Docker containers on [docker.com](https://www.docker.com/what-docker).
+
+## Requirements
+
+- `docker`
+- `docker-compose`
+
+Visit the [download page](https://www.docker.com/community-edition) for Docker tools.
+
+## Resources
+
+PHP-base images for Yii can be found at [yii2-docker](https://github.com/yiisoft/yii2-docker).
+
+
+*TBD:* Docker support for [yii2-app-advanced](https://github.com/yiisoft/yii2-app-advanced/pull/347).
+
+## Setup
+
+After installation of Docker, you should be able to run `docker ps` and see an output similar to
+
+```
+CONTAINER ID   IMAGE   COMMAND   CREATED   STATUS   PORTS
+```
+
+## Usage
+
+For a quick-start you can run the dockerized framework tests as described [here](https://github.com/yiisoft/yii2/blob/master/tests/README.md#dockerized-testing).
+
+## Advanced topics
```

Comment:
Could we add a reference to mysql? So this is more complete.

Reply:
Sure, I just didn't want to duplicate the content behind the link - since there's a MySQL example.
There's also an example in the PR for the advanced app...